andela/krypton-ah-frontend
# Component and Actions for like and dislike article #162414128

#### What does this PR do?
Create actions for getting number of reactions for an article

#### Description of Task to be completed?
Add component for reactions to read article page
Make call to the database for number of reactions and make sure respond is accurate

#### How should this be manually tested?
Git clone the repo and cd into it
npm run dev to start the server
click on an article on the homepage
open the redux dev tools to check the application action for `TOTAL_REACTIONS_SUCCESS`

#### What are the relevant pivotal tracker stories?
[Link](https://www.pivotaltracker.com/story/show/162414128)

![likedislike1](https://user-images.githubusercontent.com/22124878/53487557-2ad5d500-3a8c-11e9-9f4b-2e5ba57ee7c7.gif)
